automation/start_janssen_aio_demo.sh

```
@@ -0,0 +1,570 @@
+
+    cat > "$demo_templates_dir/configuration.json" << EOF
+{
+    "_secret": {
+        "admin_password": "Test1234#",
+        "sql_password": "Test1234#",
+        "ssl_ca_cert": "$ssl_ca_cert",
+        "ssl_ca_key": "$ssl_ca_key",
+        "ssl_cert": "$ssl_cert",
+        "ssl_key": "$ssl_key",
+        "ssl_csr": "$ssl_csr",
+        "vault_role_id": "$(cat $demo_templates_dir/vault_role_id)",
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)